hack/make-rules/verify.sh

```diff
@@ -84,6 +84,7 @@ QUICK_PATTERNS+=(
 
 EXCLUDED_CHECKS=$(ls ${EXCLUDED_PATTERNS[@]/#/${KUBE_ROOT}\/hack\/} 2>/dev/null || true)
 QUICK_CHECKS=$(ls ${QUICK_PATTERNS[@]/#/${KUBE_ROOT}\/hack\/} 2>/dev/null || true)
+TARGET_LIST=(${WHAT:-})
```

Comment:
Split string variable more robustly.

```suggestion
TARGET_LIST=()
IFS=" " read -r -a TARGET_LIST <<< "${WHAT}"
```

Reply:
I will address it.